pandas/core/generic.py

```diff
@@ -3348,12 +3354,43 @@ def reindex_like(self, other, method=None, copy=True, limit=None,
 
         Notes
         -----
-        Like calling s.reindex(index=other.index, columns=other.columns,
-                               method=...)
+        Like calling `.reindex(index=other.index, columns=other.columns,
+                               method=...)`
 
         Returns
         -------
-        reindexed : same as input
+        Object
+            same object type as input, but with changed indices on each axis
```

Comment:
You can leave out the "Object" line, and directly put the explanation on that line. 

And can you use Capital first character and end point punctuation?